server/handlers/rawCaptureHandler.js

```diff
@@ -29,15 +29,18 @@ const rawLegacyCaptureSchema = Joi.object({
   lat: Joi.number().required().min(0).max(90),
   lon: Joi.number().required().min(0).max(180),
   note: Joi.string(),
-  device_identifier: Joi.string(),
+  device_identifier: Joi.string().required(),
   planter_id: Joi.number().required(),
   planter_identifier: Joi.string().required(),
   planter_photo_url: Joi.string().uri(),
-  attributes: Joi.array(),
-  timestamp: Joi.date().timestamp('unix'),
+  attributes: Joi.array().items(Joi.object({
+    key: Joi.string().required(),
+    value: Joi.string().required()
+  })),
+  timestamp: Joi.date().timestamp('unix').required(),
 }).unknown(false);
 
-const rawCaptureGet = async (req, res, next) => {
+const rawCaptureGet = async (req, res) => {
```

Comment:
The next even though not used is needed for the global error handler the way it has been setup. So please restore the `next`. You will understand what I am talking about when you look at `routes.js` and `handlerWrapper` function that wraps all the handlers.